tests/unit/sagemaker/model/test_model.py

```diff
@@ -687,3 +728,40 @@ def test_script_mode_model_uses_proper_sagemaker_submit_dir(repack_model, sagema
         ]
         == "/opt/ml/model/code"
     )
+
+
+@patch("sagemaker.get_model_package_args")
+def test_call_to_get_model_package_args(get_model_package_args, sagemaker_session):
```

Comment:
hi mufi, I renamed it to `test_register_calls_model_package_args` because that's how the convention is in the rest of the file. So for example, another test is called: `test_script_mode_model_uses_proper_sagemaker_submit_dir`